src/routes.js

```diff
@@ -14,6 +14,7 @@ router.use(validators.authenticateRequest);
 
 // only 100 POST requests are allowed in 60 minutes window
 router.post('/*', rateLimit({ windowMs: 60 * 60 * 1000, max: 100 }), model.xpost);
+router.get('/_meta/*', model.xget_meta);
```

Comment:
Also we can skip the validators for this route by moving it above the validator middleware as this is a special route.

Reply:
Don't we still need to do _some_ validation on the `:box_id`, even if its on a special `/_meta/:box_id` route?

Reply:
I'm now checking there are records returned back for the boxId.  
So if an invalid boxId is provided the result will just be  
```json
{
    "count": 0
}
```

Reply:
I don't think we need to validate the `box_id` for this route, we show the results for a valid box, else return `0` as you have done.